authz/api/http/requests.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package http
+
+import (
+	"github.com/mainflux/mainflux/authz"
+)
+
+type addPolicyReq struct {
+	token   string
+	Subject string
```

Comment:
Are all these exported? And struct itself is not... strange.

Reply:
marshalling